Cartfile

```diff
@@ -0,0 +1,4 @@
+github "kirualex/SwiftyGif" ~> 5.0
+github "kean/Nuke" ~> 10.0
+github "daltoniam/Starscream" ~> 4.0
+github "GetStream/stream-chat-swift" ~> 4.4.0
```

Comment:
We should not need anything else than this line, given that the dependencies are listed in our library's Cartfile